data/user/src/main/kotlin/nl/q42/template/data/user/remote/model/UserDTO.kt

```diff
@@ -1,18 +1,19 @@
 package nl.q42.template.data.user.remote.model
 
-import com.squareup.moshi.JsonClass
+import kotlinx.serialization.Serializable
+
 
 /**
  * Data Transfer Models (DTO) are preferably generated from server source code/json/schemas and do not contain
  * any changes compared to the server contract.
  */
 
-@JsonClass(generateAdapter = true)
-internal data class UserDTO(
```

Comment:
Out of curiosity, why are these not internal anymore?

Reply:
Good point, this appears unnecessary. Reverted